4_extract/src/extract_model_data.R

```diff
@@ -0,0 +1,69 @@
+#' Extract model data from GLM model runs
+#'
+#' @param model_run_tbl df, a dataframe of model configuration values
```

Comment:
SC: update this function documentation to match the current version of the function